src/styles/blocks/main.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  justify-items: center;
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
```

Comment:
Your media query for `min-width: 488px` is redundant because you already set `grid-template-columns: repeat(auto-fill, 200px);` by default. So you can safely remove this media query.